package io.funwork.api.organization.application.exception;

/**
 * Created by changhwaoh on 2016. 6. 18..
 */
public class NotFoundPerson extends RuntimeException{

    public NotFoundPerson() {
        super();
    }
}
